.github/CONTRIBUTING.md

```diff
@@ -1,66 +1,87 @@
+- [Architecture](../documentation/architecture): overviews of the structure of the client, server, and shared code in this repo.
+- [FAQ](../documentation/FAQ.md): common questions about the project in general, as well as some of the technical pieces within.
+- [Resources](../documentation/resources.md): good resources for understanding this project’s tech stack.
+- [Getting started](../documentation/getting-started.md): some tools we recommend for getting the most out of this project.
 
-## Emoji commits
+### Emoji commits
 
 We have found that prefacing a commit message or PR title with an emoji can be a great way to improve the developer experience when browsing the repo code. Additionally, it is a terse way to convey information. Many of our contributors have found the guide at https://gitmoji.carloscuesta.me/ to be helpful in preserving this dynamic.
 
-## Documentation
+### Documentation
 
 If your change affects the public API of any packages within Quilt (i.e. adding or
 changing arguments to a function, adding a new function, changing the
 return value, etc), please ensure the documentation is also updated to
 reflect this. Documentation is in the `README.md` files of each package. If further documentation is needed please communicate via Github Issues.
 
-## Testing
+### Testing
 
 The packages in Quilt are used in mission-critical production scenarios. As such, we try not to merge any untested code. The coverage doesn't strictly need to be 100% across the board, but testing should remain a primary concern.
 
-## TODO Comments
+### TODO Comments
 
 TODO comments may seem like a great placeholder for work in progress. We prefer top handle this in a different way, using a combination of feature branches and github issues.
 
-### Follow-up Github issues
+#### Follow-up Github issues
 
 If your changes are complete in functionality, but you're not quite happy with auxillary things like documentation or testing, then feel free to make a github issue to track the work that needs to be done. These issues should be linked in the PRs that need a bit more work. This will allow context to be drawn from the code in a more trackable way than a TODO comment. Also, it allows the PR reviewers to see that the documentation or testing is purposefully incomplete and that an appropriate issue exists to track the follow-up work.
 
-### Feature branches
+#### Feature branches
 
 Another option, if you'd like to break work down into reviewable chunks, is to use a feature branch. This would be an initially empty branch that contains the entirety of your feature. Additional units of work can be distributed across several PRs into the feature branch, merged independently, and then the feature branch can be merged as a complete unit into the master branch, when it's ready.
 
-## Releasing a new version
+## Releasing
```

Comment:
We might want to mention that the contributor should not alter any version numbers in package.jsons